tests/phpunit/tests/test-canonical.php

```diff
@@ -459,4 +459,48 @@ public function test_sitemap_with_category() {
 			)
 		);
 	}
+
+	public function test_plain_cat_feed() {
+		// Create a category.
+		$cat_id = $this->factory->category->create(
+			array(
+				'name' => 'Test Category',
+				'slug' => 'test-category',
+			)
+		);
+		self::$model->term->set_language( $cat_id, 'en' );
+		// Create a post with the previous category.
+		$post_id = $this->factory->post->create(
+			array(
+				'post_title' => 'test',
+				'post_type'  => 'page',
+				'category'   => $cat_id,
+			)
+		);
+		self::$model->post->set_language( $post_id, 'en' );
+
+		$this->assertCanonical( '/?cat=' . $cat_id . '&feed=rss2', '/en/category/test-category/feed/' );
+	}
+
+	public function test_plain_tag_feed() {
+		// Create a tag.
+		$tag_id = $this->factory->tag->create(
+			array(
+				'name' => 'Test Tag',
+				'slug' => 'test-tag',
+			)
+		);
+		self::$model->term->set_language( $tag_id, 'en' );
+		// Create a post with the previous tag.
+		$post_id = $this->factory->post->create(
+			array(
+				'post_title' => 'test',
+				'post_type'  => 'page',
+				'tag'        => $tag_id,
+			)
+		);
+		self::$model->post->set_language( $post_id, 'en' );
+
+		$this->assertCanonical( '/?tag=test-tag&feed=rss2', '/en/tag/test-tag/feed/' );
+	}
```

Comment:
You can add the tag to the shared fixtures to and use it to keep only the assertion in the test.